library/groovy

```diff
@@ -6,36 +6,58 @@ GitRepo: https://github.com/groovy/docker-groovy.git
 
 Tags: 3.0.9-jdk8, 3.0-jdk8, jdk8
 Architectures: amd64, arm32v7, arm64v8, ppc64le
-GitCommit: e46b8fbd268e1b4559178b87358aac7385f694d2
+GitCommit: 573aabd7a3b1387db10c82806f72bed00bda2770
 Directory: jdk8
 
 Tags: 3.0.9-jdk11, 3.0-jdk11, jdk11
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: e46b8fbd268e1b4559178b87358aac7385f694d2
+GitCommit: 573aabd7a3b1387db10c82806f72bed00bda2770
+Directory: jdk11
+
+Tags: 3.0.9-jdk11-alpine, 3.0-jdk11-alpine, jdk11-alpine
+Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
+GitCommit: 573aabd7a3b1387db10c82806f72bed00bda2770
 Directory: jdk11
```

Comment:
All of the Alpine images should be pointing to a different directory. Otherwise, like the diff shows, they'll just be the Ubuntu based images tagged as "alpine".

Reply:
Good catch. That was a copy/paste error.